config/owasp/suppressions.xml

```diff
@@ -96,6 +96,20 @@
    ]]></notes>
         <packageUrl regex="true">^pkg:maven/org\.bouncycastle/bcprov\-jdk15on@.*$</packageUrl>
         <cve>CVE-2023-33202</cve>
+ </suppress>
```

Comment:
Think you can remove this now as Harpreet updated the build.gradle file

Reply:
Ok thanks have removed them